Telemetry/run.sh

```diff
@@ -15,6 +15,7 @@ start_containers() {
 
     cd $APP_HOME
     ./scripts/storageadapter.sh start
+    ./scripts/pcsconfig.sh start
```

Comment:
move this below telemetry.sh start. config should start after telemetry

Reply:
please add a comment explaining why